test/functional/mempool_accept_wtxid.py

```diff
@@ -79,10 +79,7 @@ def run_test(self):
         child_one_txid = child_one.rehash()
 
         # Create another identical transaction with witness solving second branch
-        child_two = CTransaction()
-        child_two.vin.append(CTxIn(COutPoint(int(parent_txid, 16), 0), b""))
-        child_two.vout.append(CTxOut(int(9.99996 * COIN), child_script_pubkey))
-        child_two.wit.vtxinwit.append(CTxInWitness())
+        child_two = copy.deepcopy(child_one)
```

Comment:
Importing `copy` using `import copy` at the start might solve the failing tests. It might be a better idea to import only deepcopy using `from copy import deepcopy` and then using the function directly here.
```suggestion
        child_two = deepcopy(child_one)
```
There are some other errors that I am getting in "Submit child_two to the mempool" even after making this change on my local machine.